ui/src/main/java/com/hyperwallet/android/ui/view/widget/DateParseException.java

```diff
@@ -0,0 +1,16 @@
+package com.hyperwallet.android.ui.view.widget;
+
+class DateParseException extends Exception {
```

Comment:
We actually don't need this class because of the way we handle the exception in WidgetDateDialogFragment$onCreateDialog. We can let the parse exception bubble up